src/tokens/tokens.ts

```diff
@@ -0,0 +1,16 @@
+import definitions from "./definitions.json";
```

Comment:
Do you know how this is going to work once it is compiled to JS? Is this embedded into the file?
I've made bad experiences with importing json like this, especially once you try to pack everything up with webpack (important for shipping the SDK as part of a web application).

I am not sure if this particular case would work, that is why I would avoid it in general. If the list is static, just inline it into the typescript and nothing can go wrong :)
Since it is within the SDK, the user can't modify it anyway!

Reply:
I wish I understood how MEW does it :)